code/modules/features/kada_export_feature/kada_export_feature.module

```diff
@@ -32,14 +32,14 @@ function tkufi_export_feature_views_pre_render(&$view) {
       case 'term_rss':
         if (is_numeric(arg(1))) {
           $i18n_object = i18n_get_object('taxonomy_term', arg(1));
-          $title = $i18n_object->localize($language->language)->name . ' | Turku.fi';
+          $title = $i18n_object->localize($language->language)->name . ' | example.com';
```

Comment:
This hook is used to override RSS feed and item source titles for RSS feed links found on landing pages and sections, see e.g. http://www.turku.fi/term/77/rss.xml

**Hardcoding installation-dependent strings (like originally done here) creates a future need to implement the same hook *manually* at every KADA installation.**

Instead of just replacing "Turku.fi" with "example.com", the code should be improved to find the active `domain` and its multilingual `site_name` variable.